src/main/java/uk/gov/cshr/report/controller/CourseCompletionsController.java

```diff
@@ -64,6 +64,12 @@ public AddCourseCompletionReportRequestResponse addReportRequest(@RequestBody @V
             return new AddCourseCompletionReportRequestResponse(false, "User has reached the maximum allowed report requests");
         }
         CourseCompletionReportRequest reportRequest = postCourseCompletionsReportRequestsParamsToReportRequestMapper.getRequestFromParams(params);
+
+        List<String> authorities = userAuthService.getBearerTokenFromUserAuth().getClaim("authorities");
```

Comment:
Can we interface this in the userAuthService and make it return an empty list if `getCliam` is null?